src/v0/destinations/active_campaign/transform.js

```diff
@@ -503,11 +503,13 @@ const trackRequestHandler = async (message, category, destination) => {
     );
   }
 
-  const storedEventsArr = res.response.data.eventTrackingEvents;
+  const storedEventsArr = res.response?.data?.eventTrackingEvents;
   const storedEvents = [];
-  storedEventsArr.map((ev) => {
-    storedEvents.push(ev.name);
-  });
+  if (Array.isArray(storedEventsArr)) {
+    storedEventsArr.map((ev) => {
+      storedEvents.push(ev.name);
+    });
+  }
```

Comment:
The names `storedEventsArr` and `storedEvents` are confusing. Can you rename them for avoiding it?